docs/reference/indices/open-close.asciidoc

```diff
@@ -7,6 +7,15 @@ for maintaining its metadata), and is blocked for read/write operations.
 A closed index can be opened which will then go through the normal
 recovery process.
 
+IMPORTANT: {es} will not rebuild any copies of shards of closed indices that
```

Comment:
Because there's a risk of data loss, I'd use the `WARNING` admon here.
```suggestion
WARNING: {es} will not rebuild any copies of shards of closed indices that
```